docs/source/technical_tutorials/working_with_r.rst

```diff
@@ -1,8 +1,11 @@
+.. _working_with_r_section:
```

Comment:
I had to add the reStructuredText label (reference target) to create a unique identifier we could reference in the technical tutorial rst file.